src/component/Footer.js

```diff
@@ -25,28 +24,8 @@ const Footer = () => {
         <div style={{ display: "flex", color: `${textPrimary}` }}>
           <List>
             <li>
-              <Item>Home</Item>
-            </li>
-            <li>
-              <Item>Careers</Item>
-            </li>
-            <li>
-              <Item>Blog</Item>
-            </li>
-            <li>
-              <Item>About</Item>
-            </li>
-          </List>
-          <List>
-            <li>Contact</li>
-            <li>
-              <Item>Phone: ###-###-####</Item>
-            </li>
-            <li>
-              <Item>Email: email@email.com</Item>
-            </li>
-            <li>
-              <Item>Location: Address</Item>
+              Contact:{" "}
```

Comment:
@MilioFrankie I thought the links at the bottom of the page were redundant. So, I removed them. I also only kept the single contact option `email`.

Reply:
Your right the links on the footer were redundant since the navbar is always available.